Sources/Prometheus/PrometheusMetricsFactory.swift

```diff
@@ -43,15 +43,15 @@ public struct PrometheusMetricsFactory: Sendable {
     /// The histogram buckets for a ``ValueHistogram`` per label
     public var valueHistogramBuckets: [String: [Double]]
 
-    /// A closure to modify the label and dimension names used in the Swift Metrics API. This allows users
+    /// A closure to modify the name and labels used in the Swift Metrics API. This allows users
     /// to overwrite the Metric names in third party packages.
-    public var labelAndDimensionSanitizer: @Sendable (_ label: String, _ dimensions: [(String, String)]) -> (String, [(String, String)])
+    public var nameAndLabelSanitizer: @Sendable (_ label: String, _ dimensions: [(String, String)]) -> (String, [(String, String)])
```

Comment:
the var is called nameAndLabelSanitizer but the parameters are label, dimensions
this is maybe inconsistent

Reply:
this is a bit confusing re Prometheus and Swift Metrics naming.

```
Swift Metrics -> Prometheus
label -> name
dimensions -> labels
```